contrib/inventory/gce_googleapiclient.py

```diff
@@ -43,28 +44,49 @@
 """.format(script_name=basename(argv[0]), envvar_prefix=ENV_PREFIX)
 
 
-def get_all_projects(api_version='v1'):
+
+def get_all_billing_projects(account_billing_name, api_version='v1'):
     project_ids = []
 
     credentials = GoogleCredentials.get_application_default()
 
-    service = discovery.build('cloudresourcemanager', api_version,
+    service = discovery.build('cloudbilling', 
+                              version=api_version,
                               credentials=credentials)
     # pylint: disable=no-member
-    request = service.projects().list()
+    request = service.billingAccounts().projects().list(name=account_billing_name)
 
     while request is not None:
         response = request.execute()
 
-        for project in response['projects']:
-            project_ids.append(project['projectId'])
-
         # pylint: disable=no-member
-        request = service.projects().list_next(previous_request=request,
-                                               previous_response=response)
+        request = service.billingAccounts().projects().list_next(previous_request=request,
+                                                                 previous_response=response)
+
+        for projectBillingInfo in response['projectBillingInfo']:
```

Comment:
variable name should not have UPPER CASE characters, replace `projectBillingInfo` with `project_billing_info`.
i know it was like that on the sample i've shared with you, but it was already fixed on mine.